pkg/apis/crd.nsx.vmware.com/v1alpha1/ipblocksinfo_types.go

```diff
@@ -0,0 +1,40 @@
+/* Copyright © 2024 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+//+kubebuilder:object:root=true
+//+kubebuilder:resource:scope="Cluster"
+//+kubebuilder:resource:path=ipblocksinfos
+
+// IPBlocksInfo is the Schema for the ipblocksinfo API
+type IPBlocksInfo struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// An array of CIDR strings. Each is a contiguous IP address spaces represented
```

Comment:
nit: godocs should start with the exported name.

// ExternalIPCIDRs ...

Same with PrivateTGWIPCIDRs below

Reply:
Updated, thanks!